### PR TITLE
166152332 add author field as a foreign key to Exercises 

### PR DESCRIPTION
**Pull request by martinMutuma**
#### Title
The author field should be a foreign key on exercises 
#### Description
This change is required so as to be able to get the author details for each exercise 

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue) 

- [ ] New feature (non-breaking change which adds functionality) 

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 

- [x] This change requires a documentation update

#### How Has This Been Tested?
- [x] End to end 
- [x] Unit tests

#### Checklist:
- [x] Code follows the agreed standards by the team
- [x] Code is Linted 
- [x] Add author field to exercises model as a foreign key
- [x] Add Assert statement for Author field in exercise tests

#### PT stories
[#166152332](https://www.pivotaltracker.com/story/show/166152332)
#### Screenshots 
Tested on the frontend and add exercise still works 
image
This is the entry on the database
image

### CODE REVIEW

**Review by misocho — APPROVED** [review]
The author field on the exercises model is now a foreign key and it's great you have added a test for that.

**Review by Kimaiyo077 — APPROVED** [review]
Good work on making the field to be a foreign key and adding an assertion test for the changes.

**Review by justMuriithi — APPROVED** [review]
Great job adding a test for the addition of author field as a foreign key to exercises.

File: wger/exercises/models.py [hunk 1]
```diff
@@ -213,6 +213,8 @@ class Exercise(AbstractSubmissionModel, AbstractLicenseModel, models.Model):
                             max_length=36,
                             editable=False,
                             default=uuid.uuid4)
+    Author = models.ForeignKey(User, on_delete=models.DO_NOTHING, blank=True,
```
Comment (michael-basweti) [inline wger/exercises/models.py]: I think us using `on_delete=models.DO_NOTHING` in our code will eventually create integrity issues in the case where we have deleted the user. We will be referencing to an object that doesn't exist.

**Review by michael-basweti — CHANGES_REQUESTED** [review]

File: wger/exercises/models.py (hunk 1 above)
Comment (martinMutuma) [reply wger/exercises/models.py]: Thank you for pointing that out. I have changed it to `model.SET_NULL`

**Review by Tevinthuku — APPROVED** [review]
Good job in making the author field a Foreign Key. We can now get more details from the author which is a good thing.

**Review by michael-basweti — APPROVED** [review]

**Comment by michael-basweti** [comment]
This looks good

**Pull request merged**